packages/stencil/components/select/src/index.html

```diff
@@ -26,6 +26,26 @@ <h3>default</h3>
   <button onclick="select1Toggle(event)">toggle</button>
   <p id="select1-info"></p>
 </div>
+<div style="text-align:center">
```

Comment:
Why add a new example of Select component in Tabs PR?

Reply:
I don't kwow why my bad 👎